.github/workflows/add-labels.yml

```diff
@@ -18,7 +18,12 @@ jobs:
       - name: Add labels for areas found in bug issue descriptions
         shell: pwsh
         run: |
-          $match = [regex]::Match('${{ github.event.issue.body }}', '^[#]+ Area\s*?(area:\w+)')
+          $body =
+          @"
+          ${{ github.event.issue.body }}
```

Comment:
Would this be a script injection concern? For example if the user put something like `"@` in the issue description? This bring a general concern regarding how are script injection are handled in other CI flows.

Reply:
Yikes ya you are right. I updated it so it doesn't inject anything into the script content.